src/olympia/reviewers/views.py

```diff
@@ -696,6 +696,17 @@ def review(request, addon, channel=None):
     if unlisted_only and not acl.check_unlisted_addons_reviewer(request):
         raise PermissionDenied
 
+    # Are we looking at a listed review page while only having content review
+    # permissions ? Redirect to content review page, it will be more useful.
+    if (channel == amo.RELEASE_CHANNEL_LISTED and content_review is False and
+        acl.action_allowed(request, amo.permissions.ADDONS_CONTENT_REVIEW) and
+        not any(
+            acl.action_allowed(request, perm) for perm in (
+                amo.permissions.ADDONS_REVIEW,
```

Comment:
is just `.ADDONS_REVIEW` sufficient?  i.e. does anyone actually have `.POST_REVIEW` or `.ADDONS_RECOMMENDED_REVIEW` while not also having `.ADDONS_REVIEW`?

Reply:
I'm not sure, I went for the safest approach. Eventually we want to merge `POST_REVIEW` and `ADDONS_REVIEW` for sure, but for now since they are separated everywhere in the code I treated them separately.